test/opt/test_def_use.cpp

```diff
@@ -1180,9 +1180,13 @@ ir::Instruction BranchInstruction(uint32_t target_id) {
                          });
 }
 
+// Test case for analyzing individual instructions.
 struct AnalyzeInstDefUseTestCase {
   std::vector<ir::Instruction> insts;  // instrutions to be analyzed in order.
-  InstDefUse expected_define_use;      // expected analysis result.
+  const char*
+      module_text;  // If not empty, the module will be analyzed before above
```

Comment:
Removed in c7f6a94. It is really nice that we can assemble then disassemble an empty module